modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageLayoutEditorDisplayContext.java

```diff
@@ -372,7 +372,43 @@ private JSONArray _getAssetListEntryLinkedCollectionJSONArray(
 				getGroupId(), PortalUtil.getClassNameId(Layout.class.getName()),
 				themeDisplay.getPlid(), true);
 
+		boolean addedDefault = false;
+
 		for (SegmentsExperience segmentsExperience : segmentsExperiences) {
+			if ((segmentsExperience.getPriority() <
+					SegmentsExperienceConstants.PRIORITY_DEFAULT) &&
+				!addedDefault) {
+
+				availableSegmentsExperiences.put(
```

Comment:
I think you should addedDefault to true, right?

Reply:
I don't think so

Reply:
Ahh, set addedDefault = true. Yeahhh, already done in my local code, thanks :)

Reply:
I will push it, maybe some other changes are missing as well.